applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -0,0 +1,498 @@
+				E('th', {'class': 'th'}, _('Roam src'), _('Roam source in usteer overview')),
+				E('th', {'class': 'th'}, _('Roam tgt'), _('Roam target in usteer overview'))
+			])
+		]);
+		var connectioninfo_table_entries = [];
+		this.collectVlanInfoEntries(connectioninfo_table_entries, Localinfo);
+		this.collectVlanInfoEntries(connectioninfo_table_entries, Remoteinfo);
+
+		cbi_update_table(connectioninfo_table, connectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(connectioninfo_table);
+		var compactconnectioninfo_table = E('table', {'class': 'table cbi-section-table'}, [
+			E('tr', {'class': 'tr table-titles'}, [
+				E('th', {'class': 'th'}, _('IP & Interface name', 'Combination of IP and interface name in usteer overview')),
+				E('th', {'class': 'th'}, _('SSID')),
+				E('th', {'class': 'th'}, _('Frequency', 'BSS operating frequency in usteer overview')),
+				E('th', {'class': 'th'}, _('Load', 'Channel load in usteer overview')),
+				E('th', {'class': 'th'}, _('N', 'Number of associated clients in usteer overview')),
+				E('th', {'class': 'th'}, _('Host', 'host hint in usteer overview'))
+			])
+		]);
+		var compactconnectioninfo_table_entries = [];
+		this.collectVlanInfos(Localinfo, compactconnectioninfo_table_entries, Localinfo);
+		this.collectVlanInfos(Remoteinfo, compactconnectioninfo_table_entries, Remoteinfo);
+		cbi_update_table(compactconnectioninfo_table, compactconnectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(compactconnectioninfo_table);
+		return E('div', {'class': 'cbi-section cbi-tblsection'}, [body]);
+	}
+});
+
+var Settingstitle = form.DummyValue.extend({
+	renderWidget: function () {
+		var body = E([
+			E('h3', _('Settings')),
+			E('div',
+				_('The first four options below are mandatory.') + ' ' +
+				_('Also be sure to enable rrm reports, 80211kv, etc.') + ' ' +
+				_('See <a %s>documentation</a>').format('href="https://openwrt.org/docs/guide-user/network/wifi/usteer"')
+			),
+		]);
+		return E('div', [body]);
+	}
+});
+
+var footerdata;
+var Settingsfooter = form.DummyValue.extend({
+	renderWidget: function () {
+		var body = E([
+			E('body', footerdata),
+		]);
+		return E('div', {'style': 'width:100%'}, [footerdata]);
+	}
+});
+
+
+return view.extend({
+	callHostHints: rpc.declare({
+		object: 'luci-rpc',
+		method: 'getHostHints',
+		expect: {'': {}}
+	}),
+	callGetRemotehosts: rpc.declare({
+		object: 'usteer',
+		method: 'remote_hosts',
+		expect: {'': {}}
+	}),
+	callGetRemoteinfo: rpc.declare({
+		object: 'usteer',
+		method: 'remote_info',
+		expect: {'': {}}
+	}),
+	callGetLocalinfo: rpc.declare({
+		object: 'usteer',
+		method: 'local_info',
+		expect: {'': {}}
+	}),
+	callGetClients: rpc.declare({
+		object: 'usteer',
+		method: 'get_clients',
+		expect: {'': {}}
+	}),
+	load: function () {
+		return Promise.all([
+			rpc.list('usteer'),
+			this.callHostHints().catch (function (){return null},
```

Comment:
The OpenWrt has a function that creates a promise that wraps another promise and returns a default value if it was failed: `L.resolveDefault(somePromise, [])`.
So maybe here it can be used like `L.resolveDefault(this.callHostHints, null)`.
This needs for testing so if you have a time please try.

Reply:
That does not work. It does not throw an error, but never returns anything useful.